contract-addresses/mainnetAddresses.ts

```diff
@@ -554,10 +582,6 @@ const MainnetAddresses = {
   tribe: { artifactName: 'Tribe', address: '0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B' },
   tribeOTCEscrow: { artifactName: 'OtcEscrow', address: '0xe2fE8041429e4bd51c40F92C6cDb699527171298' },
   tribeBalOtcEscrow: { artifactName: 'OtcEscrow', address: '0xfFdEe6b0261d70278f5A3093A375c282eF8266Db' },
-  tribeReserveStabilizer: {
```

Comment:
Why remove this?

Reply:
that address was never deployed. This is getting in the way of the test for some reason so I deleted